### PR TITLE
Version update for CSM 1.12

### PR DESCRIPTION
**Pull request by bjiang27**
# Description
Version update for CSM 1.12

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/1435 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

Successfully built all images with these changes
Ran cert-csi and it passed with 100%

```
[2024-10-24 14:29:32]  INFO Avg time of a run:   25.05s
[2024-10-24 14:29:32]  INFO Avg time of a del:   12.02s
[2024-10-24 14:29:32]  INFO Avg time of all:     47.31s
[2024-10-24 14:29:32]  INFO During this run 100.0% of suites succeeded
```

### CODE REVIEW

File: dell-csi-helm-installer/csi-offline-bundle.md [hunk 1]
```diff
@@ -78,13 +78,13 @@ For example, here is the output of a request to build an offline bundle for the
 *
 * Pulling and saving container images
 
-   dellemc/csi-isilon:v2.11.0
+   dellemc/csi-isilon:v2.12.0
    dellemc/csi-metadata-retriever:v1.6.0
-   dellemc/csipowermax-reverseproxy:v2.10.0
-   dellemc/csi-powermax:v2.11.0
```
Comment (adarsh-dell) [inline dell-csi-helm-installer/csi-offline-bundle.md:84]: Hi @bjiang27 , How about changing the registry reference here?
image

**Comment by adarsh-dell** [comment]
I can see that versions [here ](https://github.com/dell/csi-powermax/blob/9585cd07850eefa673697ee541a78395eb2add22/Makefile#L25)are not being updated for multiple releases. Do you think if this is required?

File: dell-csi-helm-installer/csi-offline-bundle.md (hunk 1 above)
Comment (bjiang27) [reply dell-csi-helm-installer/csi-offline-bundle.md:84]: Thank you for your comment. The changes for quay.io image updates is in progress, and will be included in a separate pull request.

**Comment by bjiang27** [comment]
> I can see that versions [here ](https://github.com/dell/csi-powermax/blob/9585cd07850eefa673697ee541a78395eb2add22/Makefile#L25)are not being updated for multiple releases. Do you think if this is required?

Thank you, updated.

**Review by abhi16394 — APPROVED** [review]

**Review by bpjain2004 — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -21,13 +21,6 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
-# Tag parameters
-MAJOR=1
-MINOR=0
-PATCH=0
-NOTES=-beta
-TAGMSG="CSI Spec 1.6"
-
```
Comment (AkshaySainiDell) [inline Makefile:30]: Could you check if these lines in docker.mk are required now ?
https://github.com/dell/csi-powermax/blob/2926bca9fe072febbc2b865ec58b63deacbabb1d/docker.mk#L19-L27

**Review by abhi16394 — APPROVED** [review]

**Review by donatwork — APPROVED** [review]

**Pull request merged**